pkg/controllers/provisioning/provisioner.go

```diff
@@ -317,21 +319,22 @@ func (p *Provisioner) Schedule(ctx context.Context) (scheduler.Results, error) {
 	s, err := p.NewScheduler(ctx, pods, nodes.Active())
 	if err != nil {
 		if errors.Is(err, ErrNodePoolsNotFound) {
-			logging.FromContext(ctx).Info(ErrNodePoolsNotFound)
+			log.FromContext(ctx).Info(ErrNodePoolsNotFound.Error())
 			return scheduler.Results{}, nil
 		}
 		return scheduler.Results{}, fmt.Errorf("creating scheduler, %w", err)
 	}
 	results := s.Solve(ctx, pods).TruncateInstanceTypes(scheduler.MaxInstanceTypes)
-	logging.FromContext(ctx).With("pods", pretty.Slice(lo.Map(pods, func(p *v1.Pod, _ int) string { return client.ObjectKeyFromObject(p).String() }), 5)).
-		With("duration", time.Since(start)).
-		Infof("found provisionable pod(s)")
+
+	log.FromContext(ctx).WithValues("pods", pretty.Slice(lo.Map(pods, func(p *v1.Pod, _ int) string { return client.ObjectKeyFromObject(p).String() }), 5)).
```

Comment:
Do you need to call `WithValues` twice? We honestly shouldn't have been calling `With` twice here for the same reason

Reply:
I have reduced it to a single `WithValue`. Please let me know if that's not what you meant.